clayui.com/static/js/code-collapse.js

```diff
@@ -0,0 +1,35 @@
+if (document.body) {
+	document.body.addEventListener('click', event => {
+		const target = event.target;
+        const targetClassList = target.classList;
+        
+		if (
+			targetClassList.contains('btn-collapse') ||
+			targetClassList.contains('icon-collapse') ||
+			(target.tagName === 'use' &&
+				target.parentNode.classList.contains('icon-collapse'))
+		) {
+			const codeContainer = target.closest('.code-container');
+            
+			if (codeContainer) {
+				const gatsbyCode = codeContainer.querySelector(
+					'code[class^="gatsby-code-"]'
+				);
+
+                if (gatsbyCode) {
+                    const collapseBtn = codeContainer.querySelector('.btn-collapse');
+
+                    if (gatsbyCode.style.display === 'none') {
+                        collapseBtn.style.transform = 'rotate(0)';
```

Comment:
Rather than rotating the icon, we should be doing the same thing that we do on the React side and just change the icon reference to angle-right